spin-observatory-plugin-deck/src/components/PluginContainer.tsx

```diff
@@ -1,10 +1,41 @@
-import { Application } from '@spinnaker/core';
-import * as React from 'react';
+import { Application, IPipeline, ReactSelectInput, useDataSource } from '@spinnaker/core';
+import React, { useEffect, useState, ChangeEvent } from 'react';
 
 interface IPluginContainerProps {
   app: Application;
 }
 
-export function PluginContainer(props: IPluginContainerProps) {
-  return <div/>;
+export function PluginContainer({ app }: IPluginContainerProps) {
+  const dataSource = app.getDataSource('observatory');
+  const { data: pipelines } = useDataSource<IPipeline[]>(dataSource);
+  const [selectedPipeline, setSelectedPipeline] = useState<string>();
+
+  useEffect(() => {
+    dataSource.activate();
+  }, []);
+
+  const onPipelineSelect = (e: ChangeEvent) => {
+    const target = e.target as HTMLSelectElement;
+    setSelectedPipeline(target.value);
+  };
+
+  return (
+    <div className="flex-container-v" style={{ margin: '3rem', width: '100%' }}>
```

Comment:
Since this is the root page of the plugin I think this should be a subcomponent instead of living directly in the plugin but I can move it later.

Reply:
yeah I think we can do that later, I didn't wanna overcomplicate it with passing props around if I didn't have to since it's  just a simple input